spec/purl_fetcher/client/upload_files_spec.rb

```diff
@@ -64,8 +64,8 @@
     end
 
     it 'returns a list of upload responses' do
-      expect(uploader.upload.count).to eq(filepath_map.count)
-      expect(uploader.upload.first).to be_a(PurlFetcher::Client::DirectUploadResponse)
+      # expect(uploader.upload.count).to eq(filepath_map.count)
+      # expect(uploader.upload.first).to be_a(PurlFetcher::Client::DirectUploadResponse)
```

Comment:
Cruft or do we want to keep these as commented out lines?

Reply:
Want to keep. I forgot to hit save in the editor. Fixed now @aaron-collier